modules/default/weather/forecast.njk

```diff
@@ -14,10 +14,10 @@
                 {% endif %}
                 <td class="bright weather-icon"><span class="wi weathericon wi-{{ f.weatherType }}"></span></td>
                 <td class="align-right bright max-temp">
-                    {{ f.maxTemperature | roundValue | unit("temperature") }}
+                    {{ f.maxTemperature | roundValue | decimalSymbol | unit("temperature") }}
```

Comment:
Not sure if the order is important here... In the current njk the decimalSymbol is applied last...

Reply:
I tried that first, but it didn't work.

Reply:
I checked it again. In the forcast part it doesn't work if `decimalSymbol` is at the end.

Reply:
How doesnt it work on your side? It looks ok if I move the dcimalSymbol to the end.

Reply:
Apparently I made the same mistake twice (insert the decimalSymbol inside the unit brackets). That couldn't work 🤦‍♂️. So it (decimalSymbol at the end) works if you do it right. Sorry.

Should I adjust my PR so that it is consistent with the current part?

Reply:
Yes, I think consistency is king :-) Have a good monday!

Reply:
I changed it :slightly_smiling_face: